etc/config.xml

```diff
@@ -18,6 +18,7 @@
                 <mode>live</mode>
                 <is_auth>0</is_auth>
                 <allowed_payment_methods>creditCard,bank,payWithTerms,invoice</allowed_payment_methods>
+                <allowed_customer_groups>1,2,3</allowed_customer_groups>
```

Comment:
Just making sure – groups 1-3 will always exist?

Reply:
Yes those are default Magento customer groups, which will be available.